16x9/DialogFullScreenInfo.xml

```diff
@@ -5,6 +5,7 @@
 	<onunload>Skin.Reset(PausedOverlayCast)</onunload>
 	<onunload>Skin.Reset(ShowPausedOverlayFlags)</onunload>
 	<onunload>Skin.Reset(ShowSeekbar)</onunload>
+	<onunload condition="VideoPlayer.Content(livetv) + !Skin.HasSetting(ToggleLiveTVNext)">Skin.ToggleSetting(ToggleLiveTVNext)</onunload>
```

Comment:
No problem buddy 😃... Skin. Reset(The setting to reset) an example is [HERE](https://github.com/mikesilvo164/Aeon-Nox-SiLVO/blob/master/16x9/DialogFullScreenInfo.xml#L5)